core/src/trezor/messages.py

```
@@ -8855,6 +8872,44 @@ def __init__(
         def is_type_of(cls, msg: Any) -> TypeGuard["TronUnDelegateResourceContract"]:
             return isinstance(msg, cls)
 
+    class TronCancelAllUnfreezeV2Contract(protobuf.MessageType):
+
+        @classmethod
+        def is_type_of(cls, msg: Any) -> TypeGuard["TronCancelAllUnfreezeV2Contract"]:
```

Ruff (0.14.5)

Dynamically typed expressions (typing.Any) are disallowed in `msg`

(ANN401)